app/controllers/hyrax/dashboard/collections_controller_decorator.rb

```diff
@@ -19,21 +20,14 @@ def show
         super
       end
 
-      # OVERRIDE Hyrax v5.0.0 to add the ability to upload a collection thumbnail - START
-      def process_branding
-        process_banner_input
-        process_logo_input
-        process_thumbnail_input
-      end
-
       # rubocop:disable Metrics/AbcSize
       def update_valkyrie_collection
         return after_update_errors(form_err_msg(form)) unless form.validate(collection_params)
 
         result = transactions['change_set.update_collection']
                  .with_step_args(
                           'collection_resource.save_collection_banner' => { update_banner_file_ids: params["banner_files"],
-                                                                            banner_unchanged_indicator: params["banner_unchanged"] },
```

Comment:
banner_unchanged was unneeded, because we need the ability to save text even if the banner doesn't change.